iceoryx_posh/include/iceoryx_posh/popo/condition.hpp

```diff
@@ -40,12 +40,12 @@ class Condition
   private:
     friend class WaitSet;
     /// @brief User interface for specific attach of condition variable
-    virtual bool setConditionVariable(ConditionVariableData* const ConditionVariableDataPtr) noexcept = 0;
+    virtual bool setConditionVariable(ConditionVariableData* const conditionVariableDataPtr) noexcept = 0;
```

Comment:
do we really want this notation with `Ptr` postfix?

Reply:
We have this everywhere in the new building blocks, so I decided to stick to the name to be consistent. No hard feels on this, I don't mind the `Ptr` abbrevation too much.